src/screens/ContactScreen/elements/ItemUser.js

```diff
@@ -60,14 +66,22 @@ const ItemUser = ({photo, username, followed, onPress, userid}) => {
   );
 };
 
+ItemUser.propTypes = {
+  photo: PropTypes.string,
+  username: PropTypes.string,
+  followed: PropTypes.any,
+  onPress: PropTypes.func,
+  userid: PropTypes.string,
+  isAnon: PropTypes.bool
+};
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Adding PropTypes for type checking is a good practice to ensure the correct types of props are passed to the component. The addition of the `isAnon` propType is consistent with the new changes.
```diff
+ ItemUser.propTypes = {
+   photo: PropTypes.string,
+   username: PropTypes.string,
+   followed: PropTypes.any,
+   onPress: PropTypes.func,
+   userid: PropTypes.string,
+   isAnon: PropTypes.bool
+ };
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->